devspace.yaml

```diff
@@ -69,7 +75,6 @@ dev:
         - '!/cmd'
         - '!/vendor'
         - '!/hack'
-        - '!/manifests'
```

Comment:
Would be nice to keep this^
We can add `- '/manifests/coredns'`on the next line to fix the issue with the mounted ConfigMap volume being Readonly.